ApiEverest/CrudApi/Controllers/CostumerController.cs

```diff
@@ -0,0 +1,96 @@
+﻿using AppModels;
+using AppServices.Services;
+using AutoMapper;
+using DomainModels.Entities;
+using DomainServices.Services;
```

Comment:
```suggestion
using AutoMapper;
using DomainModels.Entities;
using DomainServices.Services;
```
Remover usings não utilizados. CTRL + R + G